php/async/Exchange.php

```diff
@@ -96,6 +96,7 @@ public function fetch($url, $method = 'GET', $headers = null, $body = null) {
                 include_once ($this->proxy_files_dir. 'reactphp-http-proxy/src/ProxyConnector.php');
                 $proxy = new \Clue\React\HttpProxy\ProxyConnector($httpProxy);
                 $request_browser_options = array( 'tcp' => $proxy, 'dns' => false );
+                $this->set_request_browser($request_browser_options);
```

Comment:
my bad, this line was missing in php from `httpProxy` (as opposed to `httpsProxy`)